src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -130,6 +130,9 @@ private void SetException(Exception e)
         //
         private void CloseInternal()
         {
+            if (NetEventSource.IsEnabled)
```

Comment:
nit: In most of our sources, these simple trace statements are on a single line:

```c#
if (NetEventSource.IsEnabled) NetEventSource.Enter(this);
```

Reply:
That's contrary to the coding guidelines for the repo. I disliked the unbraced style so I checked and unbraced multi line if is permitted but single line is not. See https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md item 17.